woocommerce/api/Abstract_Cacheable_API_Base.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace SkyVerge\WooCommerce\PluginFramework\v5_10_10;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_10_10\API\Cacheable_Request_Trait;
+
+abstract class Abstract_Cacheable_API_Base extends SV_WC_API_Base
```

Comment:
At a minimum let's make sure and add a `@since` tag for this, and a description if one is useful